wrappers/otp.rb

```diff
@@ -87,7 +87,7 @@ def route(locs, dimension, departure, arrival, language, with_geometry, options
           type: 'Feature',
           properties: {
             router: {
-              total_distance: i['walkDistance'] || 0, # FIXME walk only
+              total_distance: i['legs'].map{ |leg| leg['distance'] }.reduce(&:+),
```

Comment:
Do the `legs` include the `walkDistance`? or should we add it to legs? For instance, if no leg, the distance is the flying distance, isn't it?

I would add a `walk_distance` in output, for fallback in case a customer needs it. The entity `RouteResultPropertiesRouter` should contains in this case:
```
# OPTIONAL
expose_not_nil(:walk_distance, documentation: { type: Float, is_array: true, desc: 'Walk distance in meters.' })
```
after `total_distance`

Reply:
Yes, the `legs` include the `walkDistance` by foot. With no leg, OTP returns nothing.

I'll add the walk_distance to the Result. Nevertheless, walk_distance should not be an array if you want it as a fallback